core/new-gui/src/app/common/service/user/user.service.ts

```diff
@@ -42,6 +42,10 @@ export class UserService {
     return this.currentUser !== undefined;
   }
 
+  public isAdmin(): boolean {
+    return this.currentUser?.role == Role.ADMIN;
```

Comment:
Always prefer `===` in TypeScript. See this[ post](https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons).

Reply:
fixed.

Reply:
Please fix the other place too.

Reply:
fixed.